### PR TITLE
Bw2

### PR DESCRIPTION
**Pull request by sebffischer**

### CODE REVIEW

File: book/chapters/chapter3/evaluation_and_benchmarking.qmd [hunk 1]
```diff
@@ -248,46 +248,38 @@ While we are usually interested in this aggregated score, it can be useful to lo
...
-#| message: false
-plt = ggplot2::last_plot()
-plt$layers[[1]]$aes_params$fill = "white"
-print(plt)
-```
 
 ```{r}
-#| output: false
-#| cache: false
+#| eval: false
```
Comment (RaphaelS1) [inline book/chapters/chapter3/evaluation_and_benchmarking.qmd:257]: Should this really be eval: false ?

File: book/chapters/chapter3/evaluation_and_benchmarking.qmd [hunk 2]
```diff
@@ -874,50 +852,43 @@ These visualizations are also available for `r ref("ResampleResult")` objects.
 In this case, the predictions of individual resampling iterations are merged before calculating a ROC or PR curve (micro averaged):
 
 ```{r performance-058}
-#| output: false
-#| cache: false
+#| eval: false
```
Comment (RaphaelS1) [inline book/chapters/chapter3/evaluation_and_benchmarking.qmd:855]: should this be eval: false?

File: book/chapters/chapter3/evaluation_and_benchmarking.qmd (hunk 1 above)
Comment (sebffischer) [reply book/chapters/chapter3/evaluation_and_benchmarking.qmd:257]: yes, (all the code is run in the previous chunk)

File: book/chapters/chapter3/evaluation_and_benchmarking.qmd (hunk 2 above)
Comment (sebffischer) [reply book/chapters/chapter3/evaluation_and_benchmarking.qmd:855]: yes, the code is actually evaluated in the next chunk

**Review by RaphaelS1 — APPROVED** [review]

**Pull request merged**